article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -11,7 +11,7 @@
     <div
         id="block-@block.id"
         data-sort-time="@block.publishedCreatedTimestamp()"
-        class="block block--content@block.eventClass"
+        class="block block--content@block.eventClass@block.attributes.membershipPlaceholder.map { _ => js-insert-epic-after}"
```

Comment:
As discussed in person, none of our JS can run on AMP so the class isn't doing anything so might not be worth having